source/assets/stylesheets/locastyle/base/_color-theme.sass

```diff
@@ -337,13 +337,13 @@
 
       tbody
         tr:hover,
-        tr.ls-selected
+        tr.is-selected
           +rgba(background-color, $color2, .02)
         a:not([class*="ls-btn"]),
           color: $color1
 
       &.ls-no-hover
-        tbody tr.ls-selected:hover
+        tbody tr.is-selected:hover
```

Comment:
The right name of this class is not **ls-is-selected**?
We always use the prefix **ls-**, even in status classes like **selected**. And yes, the classes **ls-active** founded in another places in framework are wrong. The right is **ls-is-active**.